projects/aws/image-builder/CHECKSUMS

```diff
@@ -1,2 +1,2 @@
 762affa87b33e19a603c89786e45f684a8d8fe232e76b7aec68e38cc3632aa9b  _output/bin/image-builder/linux-amd64/image-builder
-33cd605993b69d713e959f4d1d2e7a19a6c389d058a66f2e038ad282122d4929  _output/bin/image-builder/linux-arm64/image-builder
\ No newline at end of file
+33cd605993b69d713e959f4d1d2e7a19a6c389d058a66f2e038ad282122d4929  _output/bin/image-builder/linux-arm64/image-builder
```

Comment:
These checksums seem to be wrong, Can we change them to the ones listed [here](https://prow.eks.amazonaws.com/view/s3/prowpresubmitsdataclusterstack-prowbucket7c73355c-vfwwxd2eb4gp/pr-logs/pull/aws_eks-anywhere-build-tooling/1332/aws-image-builder-presubmit/1575864608487378944#2:build-container-build-log.txt%3A185)